bundles/org.openhab.binding.mqtt.espmilighthub/README.md

```diff
@@ -152,6 +154,14 @@ For example:
 | 0xB4CA    | 8        | 0xB4CA8  |
 | 0xB4CA    | 0        | 0xB4CA0  |
 
+## Using the ALL group
+
+The ALL group with the Group ID 0 can be used to control all bulbs that are paired with one specific remote at once. While this functionality can also be achieved by using OpenHab groups with even greater flexibility the ALL group must be setup if you want to capture remote events for the ALL group to keep physical devices synchronized to their OpenHab representations. Milight remotes send all commands with the Group ID 0 after the master ON/OFF buttons have been used. If the ALL group has not been setup these events will be lost and your Item states will no longer be synchonized with the actual device states until you issue a command via OpenHab. If you do not use a remote at all or you only control other bulbs than the ones controlled by OpenHab you should not need to setup the ALL group.
+
+Since the ALL group is not needed in every case the autodiscovery feature will not detect this group as a Thing automatically. To create the group manually proceed as described in section [Important for Textual Configuration](#important-for-textual-configuration) or use the OpenHab 3 MainUI for this purpose. To create a Thing for the ALL group via MainUI simply create a new Thing that has the same type as one of autodiscovered Things of the same remote and modify the ThingUID as described in section linked above (replace the trailing 1-digit number of an autodiscovered ThingUID with a 0).
+
+Note that the behaviour of the ALL group Thing is special: After the Thing has been successfully setup all remote events with Group ID 0 are automatically also fired on all other Things that belong to the ALL group and thus all Items linked to these Things will be updated as well. While it is technically possible to setup separate Items for the ALL group itself this should not be required for most users.
```

Comment:
> While it is technically possible to setup separate Items for the ALL group itself this should not be required for most users.<

Suggest this is re-written to explain why it is not required. Something like.

If you do not need separate group 0 controls in openHAB, but wish to have all the controls for the sub groups update when a physical remote is used, you only need to create the thing for group 0.
Only if you want the controls do you need to link any channels and create the items, as creating the thing will subscribe the binding to the MQTT topic for group 0.

Reply:
I replaced my text with your suggestion and in addition to that I also primarily use the term "group 0" instead of "ALL group" in the rest of the PR.